apps/darts-modernisation/darts-api/darts-api.yaml

```diff
@@ -1,19 +1,24 @@
+---
 apiVersion: helm.toolkit.fluxcd.io/v2beta1
 kind: HelmRelease
 metadata:
   name: darts-api
+  namespace: darts-modernisation
 spec:
   releaseName: darts-api
-  values:
-    java:
-      replicas: 1
-      useInterpodAntiAffinity: true
-      image: sdshmctspublic.azurecr.io/darts/api:prod-8af60e4-20230324100423 #{"$imagepolicy": "flux-system:darts-api"}
   chart:
     spec:
       chart: ./stable/darts-api
       sourceRef:
         kind: GitRepository
         name: hmcts-charts
         namespace: flux-system
-      interval: 8m
+      interval: 1m
+  values:
+    java:
+      image: sdshmctspublicsbox.azurecr.io/darts/api:latest # {"$imagepolicy": "flux-system:darts-api"}
+      ingressHost: darts-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
this shouldn't be set here, I've fixed in the script: https://github.com/hmcts/sds-flux-config/pull/2817
```suggestion
```